node/pkg/watchers/evm/watcher.go

```diff
@@ -314,6 +318,19 @@ func (w *Watcher) Run(ctx context.Context) error {
 				}
 
 				for _, msg := range msgs {
+					if msg.ConsistencyLevel == magicNumberPublishImmediately {
+						logger.Info("re-observed message publication transaction, publishing it immediately",
```

Comment:
maybe use the same log message as below and add a new field for consistency level?

Reply:
I actually like having a different message that I can grep for, so I would like to keep this as is.